docs/releasenotes/unreleased/rules.7.rst

```diff
@@ -0,0 +1,14 @@
+New rule arguments-per-continuation-line (#1000)
+------------------------------------------------
+
+Added new I0532 ``arguments-per-continuation-line`` rule.
```

Comment:
What about just `arguments-per-line`?

Reply:
It does sound better. Also now I realized I'm only checking from the second line onwards. But I need to check it from the first line if the arguments settings is multiline. I will fix it